src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -260,22 +262,42 @@ private void saveUser(User entity, Map<String, String> okapiHeaders,
     });
   }
 
-  private boolean isDuplicateIdError(AsyncResult<Response> reply) {
```

Comment:
These methods had to be changed because PgUtil class returns errors in different format now.
Additionally PUT /users endpoint doesn't have 422 error, so logic for PUT and POST endpoints is now different